events/python/receive_pika.py

```diff
@@ -0,0 +1,24 @@
+import pika
+
+from setup_pika import (
+    DASHBOARD_QUEUE_NAME,
+    get_channel
+)
+
+
+def message_handler(channel: pika.channel.Channel, method: pika.spec.Basic.Deliver,
+                    properties: pika.spec.BasicProperties, body: str) -> None:
+    print(f'[x] Received: {body}')
+    channel.basic_ack(method.delivery_tag)
```

Comment:
same as for the other example, `no_ack=False` makes this unnecessary or is the other way around?

Reply:
(could mean either "the library should not ack because the handler will do it" or "the handler should not have to ack")

Reply:
I found through later testing that if I did not manually `ack` in the handler then the messages were labelled as received but `unacked` by the server.

Reply:
Good to know